recipes/fault-localization/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "fault-localization" %}
+{% set version = "0.1.3" %}
+{% set file_ext = "tar.gz" %}
```

Comment:
We need to fix skeleton and the example to avoid this excess of jinja vars :-(

Reply:
Yeah the `file_ext` seems a bit too much, specially that PyPI is planning to standardize on `tar.gz` only.